<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_frequent_event">
    <property name="title">
     <string>add frequent event</string>
    </property>
    <addaction name="actionEveryday"/>
    <addaction name="separator"/>
    <addaction name="actionEveryweek"/>
    <addaction name="actionEverymonth"/>
   </widget>
   <widget class="QMenu" name="menuFix_menu">
    <property name="title">
     <string>Fix menu</string>
    </property>
    <addaction name="actionFix"/>
   </widget>
   <addaction name="menuAdd_frequent_event"/>
   <addaction name="menuFix_menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEveryday">
   <property name="text">
    <string>everyday</string>
   </property>
  </action>
  <action name="actionEveryweek">
   <property name="text">
    <string>everyweek</string>
   </property>
  </action>
  <action name="actionEverymonth">
   <property name="text">
    <string>everymonth</string>
   </property>
  </action>
  <action name="actionEveryyear">
   <property name="text">
    <string>everyyear</string>
   </property>
  </action>
  <action name="actionFix">
   <property name="text">
    <string>Fix</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
